docs/daily/2025-01-03.md

```
@@ -0,0 +1,340 @@
+## ✨ 과제 리뷰
+
+데이터 받는 옵션 추가
+
+```kotlin
+@OptIn(ExperimentalSerializationApi::class)
+private val json = Json { allowTrailingComma = true }
+```
+
+```kotlin
+val json: Json = Json {
+	serializersModule = SerializersModule {
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)